ui/js/page/settings/view.jsx

```diff
@@ -199,6 +217,41 @@ class SettingsPage extends React.PureComponent {
               )}
             </div>
           </div>
+          <div className="card__content">
+            <div className="form-row">
```

Comment:
These should probably be `FormRow` components, and definitely need labels.

Reply:
I converted the "disabled" option to a `FormRow`. For the second option (where you choose the actual setting), I left it as a div for now. If made it a `FormRow`, we would have to stufff the price selector inside the label prop, which feels awkward. The Max Key Fee section also uses a div, I'm guessing for the same reason. Let me know if you'd rather it be inside the label, it's very quick fix.

I added helper text in the most recent commit; let me know if you have any adjustments on the wording.